wmn-data.json

```diff
@@ -1486,36 +1541,118 @@
+        "post_body" : "{\"username\": \"{account}\"}",
+        "e_code" : 200,
+        "e_string" : "\"taken\":true",
+        "m_string" : "\"taken\":false",
+        "m_code" : 200,
+        "known" : ["test", "web"],
+        "cat" : "social",
+        "headers" : {
+            "Content-Type" : "application/json"
+        }
+       },
+       {
+        "name" : "Discord Invites",
+        "uri_check" : "https://discord.com/api/v9/invites/{account}?with_counts=true&with_expiration=true",
+        "uri_pretty" : "https://discord.com/invite/{account}",
+        "e_code" : 200,
+        "e_string" : "\"channel\":",
+        "m_string" : "\"message\": \"Unknown Invite\"",
+        "m_code" : 404,
+        "known" : ["test", "web"],
+        "cat" : "social"
+       },
+       {
+        "name" : "DOU",
+        "uri_check" : "https://dou.ua/users/{account}/",
+        "e_code" : 200,
+        "e_string" : "class=\"page-profile\"",
+        "m_string" : "class=\"page-error\"",
+        "m_code" : 404,
+        "known" : ["doucommunity", "volodymyrobrizan"],
+        "cat" : "social"
+       },
+       {
+        "name" : "Docker Hub Users",
         "uri_check" : "https://hub.docker.com/v2/users/{account}/",
+        "uri_pretty" : "https://hub.docker.com/u/{account}",
         "e_code" : 200,
-        "e_string" : "uuid",
-        "m_string" : "User not found",
+        "e_string" : "\"uuid\":",
+        "m_string" : "\"message\":\"User not found\"",
         "m_code" : 404,
-        "known" : ["bitnami", "torvalds"],
+        "known" : ["dannapierskitoptal", "torvalds"],
+        "cat" : "coding"
+       },
+       {
+        "name" : "Docker Hub Organizations",
+        "uri_check" : "https://hub.docker.com/v2/orgs/{account}/",
+        "uri_pretty" : "https://hub.docker.com/u/{account}",
+        "e_code" : 200,
+        "e_string" : "\"uuid\":",
+        "m_string" : "\"orgname\":[\"",
+        "m_code" : 404,
+        "known" : ["bitnami", "tensorflow"],
         "cat" : "coding"
        },
        {
         "name" : "Donation Alerts",
         "uri_check" : "https://www.donationalerts.com/api/v1/user/{account}/donationpagesettings",
         "uri_pretty" : "https://www.donationalerts.com/r/{account}",
         "e_code" : 200,
-        "e_string" : "background_image_url",
+        "e_string" : "\"data\":",
         "m_code" : 202,
-        "m_string" : "does not exist",
+        "m_string" : "\"success\":false",
         "known" : ["gorou", "saku"],
-        "cat" : "business"
+        "cat" : "finance"
+       },
+       {
+        "name" : "Donatty",
+        "uri_check" : "https://api.donatty.com/users/find/{account}",
+        "uri_pretty" : "https://donatty.com/{account}",
+        "e_code" : 200,
+        "e_string" : "\"response\":",
+        "m_string" : "\"error\":\"internal error\"",
+        "m_code" : 404,
+        "known" : ["takaisekai", "takaisekai"],

```

Comment:
Looks like you have a duplicate username here for known. please choose 2 different users.